README.md

```diff
@@ -37,6 +45,10 @@ var add = function (x, y) {
 
 -- YOUR ANSWER HERE --
 
+- ES5
+- Is the function anonymous? NO
```

Comment:
It is anonymous. This is a **function expression**

```var add = function addition(x,y) {
    return x+y;
}
```

Would be the non-anonymous equivalent